emails/2022-08-05.md

```diff
@@ -1,17 +1,25 @@
-
+Light summer week this week. Lots of people on vacation. Here's the update.
```

Comment:
Intro yourself, assume someone is reading for the first time. maybe make a joke for the people that read every week?

Reply:
Better?

Reply:
love it